File: Kotlin/ROUND_2_A/src/main/kotlin/FootballData.kt


object FootballData {
    val allPlayed: List<Game>
        get() {
            val played = listOf(
            Game("Liverpool", 2, "Everton", 0),
            Game("Aston Villa", 1, "Liverpool", 1),
            Game("Liverpool", 3, "West Ham", 1),
            )
            return played
        }
}

File: csharp/ROUND_2/ROUND_2_B/src/FootballData.cs
namespace ROUND_2_B;

public class FootballData : IGameData
{
    public Game[] GetAllPlayed()
    {
        Game[] played =
        [
            new Game("Liverpool", 2, "Everton", 0),
            new Game("Aston Villa", 1, "Liverpool", 1),
            new Game("Liverpool", 3, "West Ham", 1)
        ];
        return played;
    }
}